scripts/TEST.vcf

```diff
@@ -0,0 +1,500 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##FILTER=<ID=Consensus,Description="Called by two or more callers">
+##FILTER=<ID=Hotspot,Description="In mutational hotspot, as defined by ????">
+##INFO=<ID=MQ,Number=1,Type=Integer,Description="RMS mapping quality (normal sample)">
+##INFO=<ID=MQ0,Number=1,Type=Integer,Description="Number of MAPQ=0 reads (normal sample)">
+##INFO=<ID=CAL,Number=.,Type=String,Description="List of callers making this call">
+##FORMAT=<ID=AGT,Number=.,Type=String,Description="Consensus or majority genotype">
+##FORMAT=<ID=GTC,Number=.,Type=String,Description="Genotypes for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=GT_STATUS,Number=.,Type=String,Description="Degree of unanimity of genotype">
+##FORMAT=<ID=ADC,Number=.,Type=String,Description="Allele depths for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=DPC,Number=.,Type=String,Description="Read depths for Strelka2|Mutect2|VarDict|Lancet">
+##FORMAT=<ID=AFC,Number=.,Type=String,Description="Allele frequencies for Strelka2|Mutect2|VarDict|Lancet">
+##contig=<ID=chr1,length=248956422>
+##contig=<ID=chr2,length=242193529>
+##contig=<ID=chr3,length=198295559>
+##contig=<ID=chr4,length=190214555>
+##contig=<ID=chr5,length=181538259>
+##contig=<ID=chr6,length=170805979>
+##contig=<ID=chr7,length=159345973>
+##contig=<ID=chr8,length=145138636>
+##contig=<ID=chr9,length=138394717>
+##contig=<ID=chr10,length=133797422>
+##contig=<ID=chr11,length=135086622>
+##contig=<ID=chr12,length=133275309>
+##contig=<ID=chr13,length=114364328>
+##contig=<ID=chr14,length=107043718>
+##contig=<ID=chr15,length=101991189>
+##contig=<ID=chr16,length=90338345>
+##contig=<ID=chr17,length=83257441>
+##contig=<ID=chr18,length=80373285>
+##contig=<ID=chr19,length=58617616>
+##contig=<ID=chr20,length=64444167>
+##contig=<ID=chr21,length=46709983>
+##contig=<ID=chr22,length=50818468>
+##contig=<ID=chrX,length=156040895>
+##contig=<ID=chrY,length=57227415>
+##contig=<ID=chrM,length=16569>
```

Comment:
Do we drop the contigs in consensus?

Reply:
Hmmm, I given the interval lists we give, I don't think the source files actually have calls outside of the canonical chromosomes. Those caller are slightly misleading and probably inherited from the bam file itself

Reply:
I set it up to drop all non-canonical chromosomes because I thought you had mentioned wanting that, but can add back if desired.